app-admin/opendoas/spec

```diff
@@ -0,0 +1,3 @@
+VER=6.8.2
+SRCS="tbl::https://github.com/Duncaen/OpenDoas/archive/refs/tags/v$VER.tar.gz"
+CHKSUMS="sha256::6da058a0e70b7543bc60624389b0b00b686189ec933828c522bf8b2600495a67"
```

Comment:
```suggestion
CHKSUMS="SKIP"
```

source code hosting platform may regenerate code archive, causing checksum mismatch.

also add an empty line before end of file

Reply:
for the empty line, I would.